docs/Home.md

```diff
@@ -5,24 +5,21 @@ effective use of the Brim desktop application and related tools.
 
 > **Note:** In Brim release `v0.25.0` and newer, functionality related to
 > pcap processing (including the generation of Zeek and Suricata summary logs
-> from pcaps) is now handled by [Brimcap](https://github.com/brimdata/brimcap).
-> Brimcap is bundled with Brim to provide a pcap import/extraction workflow
-> similar to what was present in Brim versions `v0.24.0` and older. A separate
-> [Brimcap wiki](https://github.com/brimdata/brimcap/wiki) exists that contains
-> current docs related to pcap processing.
+> from pcaps and pcap import/extraction workflows) is now handled by
+> [Brimcap](https://github.com/brimdata/brimcap), which is bundled with Brim.
+> A separate [Brimcap wiki](https://github.com/brimdata/brimcap/wiki)
+> contains current docs related to pcap processing.
 
 ## Support Resources
 
 - [[Installation]]
 - [[Filesystem Paths]]
 - [[Supported Platforms]]
-- [[Microsoft Windows beta limitations]]
 - [[Troubleshooting]]
 
 ## User Documentation
 
 - [[Migration for Version 0.29]]
-- [[Linux RPM Upgrade]]
```

Comment:
Here's a little history about this RPM article and an explanation of why I propose getting rid of it.

The new installer that arrived with Brim `v0.25.0` marked a point where users of the RPM packages had to manually uninstall the `v0.24.0` that used the old installer before the new version would install successfully, since otherwise there'd be errors about "conflicts". At the time I mistakenly thought that was a one-time problem specific to the point of transition between installers. However, testing with more recent releases has shown me that the problem still shows up between more recent releases that all use the new installer. Therefore, in this PR I've moved the uninstall guidance to the main Installation article so the standalone RPM article can be removed.

Furthermore, I took the opportunity to take a closer look at the experience with the newer installer and DEB packages. While this doesn't have the "conflict" problem that the RPM package has and the install appears to complete successfully, if I do the install from the CLI, it explains that this effectively installs the new version _atop_ the older one. This probably causes no real problems, but it seems messy because it has the potential for "droppings" to be left behind in old installs that are no longer referenced in newer ones. That might create confusion in debug scenarios.

Therefore, I chose to keep it simple in the Installation article and recommend an uninstall before upgrade on all Linux platforms.